### PR TITLE
Add supporting/documentation files

### PR DESCRIPTION
**Pull request by blairconrad**
Supports FakeItEasy/FakeItEasy#1735.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+[![NuGet version](https://img.shields.io/nuget/v/FakeItEasy.Analyzers.svg?style=flat)](https://www.nuget.org/packages/FakeItEasy)
+[![Build status](https://ci.appveyor.com/api/projects/status/tmxobysgprwpecsb/branch/master?svg=true)](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy-analyzers/branch/master)
+
+Roslyn analyzers to detect incorrect usages of FakeItEasy.
+
+Add an analyzer to your projects to avoid, detect, and fix common programming errors when working with FakeItEasy.
+Analyzers exist for both C# and VisualBasic.
+* [Documentation](https://fakeiteasyanalyzers.readthedocs.io/en/stable/)
+* [Chat](https://gitter.im/FakeItEasy/FakeItEasy)
```
Comment (blairconrad) [inline README.md]: _should_ we advertise a FakeItEasy/FakeItEasy.Analyzers chat?

File: how_to_build.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    `build.cmd -s pack`
+
+* View the full list of options:
+
+    `build.cmd -?`
+
+(On Linux, just replace `build.cmd` with `./build.sh`)
+
+### Building only a subset of the supported target frameworks
```
Comment (blairconrad) [inline how_to_build.md]: It is worth keeping the profiles for the analyzers? They build and test pretty fast (for now).

File: README.md (hunk 1 above)
Comment (thomaslevesque) [reply README.md]: No, I think we should use the main chat. No point in having a separate one.

File: how_to_build.md (hunk 2 above)
Comment (thomaslevesque) [reply how_to_build.md]: Probably not. I had not included this in my attempt to move the analyzers.

**Comment by thomaslevesque** [comment]
You'll need to rebase before I can merge

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
Thank you!

**Pull request merged**